docs/content/docs/concepts/additional-context/context-attachments.mdx

```diff
@@ -1,18 +1,18 @@
 ---
 title: Context Attachments
-description: Add visual context badges that automatically provide AI with focused information about what users are working on.
+description: Add visual context badges above the message input for user feedback
 ---
 
-Context attachments enable you to display visual badges above the message input that automatically provide the AI with additional context. This helps the AI understand what the user is focused on and provide more relevant responses.
+Context attachments enable you to display visual badges above the message input.
```

Comment:
yeah all of the language around this makes me feel like this is purely a visual thing, which is odd considering it's part of a non-visual sdk